SaltwaterTaffy/Ndiff.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+
+namespace SaltwaterTaffy
+{
+    public static class Ndiff
+    {
+        /// <summary>
+        ///     Run the Ndiff tool to compare output files from Nmap.
+        /// </summary>
+        /// <param name="file1">Original File</param>
+        /// <param name="file2">Modified File</param>
+        /// <param name="verbose">Also show hosts and ports that haven't changed.</param>
+        /// <param name="xml">Display output in XML format.</param>
+        /// <returns>Output from Ndiff application.</returns>
+        public static string RunDiff(bool verbose, bool xml, string file1, string file2)
```

Comment:
Why not name this `Run`?

Reply:
I honestly do not know. I'll change to Run.